jme3-core/src/main/java/com/jme3/texture/FrameBuffer.java

```diff
@@ -249,6 +248,40 @@ public static FrameBufferTextureTarget newTarget(Texture tx, TextureCubeMap.Face
             return t;
         }
     }
+    
+    /**
+     * Quick access method for creating a texture target.
+     * 
+     * @param tx
+     * @return 
+     * @see FrameBufferTarget#newTarget(com.jme3.texture.Texture)
+     */
+    public static FrameBufferTextureTarget target(Texture tx) {
+        return FrameBufferTarget.newTarget(tx);
+    }
+    
+    /**
+     * Quick access method for creating a buffer target of the specified format.
+     * 
+     * @param format
+     * @return 
+     * @see FrameBufferTarget#newTarget(com.jme3.texture.Image.Format)
+     */
+    public static FrameBufferBufferTarget target(Format format) {
```

Comment:
This method seems redundant and unnecessary to me, please remove it. 

If you need it, use the static `FrameBufferTarget.newTarget(Format)` method directly.